albumentations/augmentations/functional.py

```diff
@@ -1793,3 +1793,22 @@ def superpixels(
 @clipped
 def add_weighted(img1, alpha, img2, beta):
     return img1.astype(float) * alpha + img2.astype(float) * beta
+
+
+@clipped
+@preserve_shape
+def unsharp_mask(image: np.ndarray, ksize: int, sigma: float = 0.0, alpha: float = 0.2, threshold: int = 10):
+    blur_fn = _maybe_process_in_chunks(cv2.GaussianBlur, ksize=(ksize, ksize), sigmaX=sigma)
+    blur = blur_fn(image)
+    residual = image - blur
```

Comment:
We must do this operation in int32 or float data type, I am not sure that it will be automatically converted to the correct data type